tests/epyccel/test_epyccel_division.py

```diff
@@ -144,7 +144,7 @@ def fdiv_i_i(x, y):
     y = randint(low=1, high= 100, dtype='int8')
 
     assert (f(x, y) == fdiv_i_i(x, y))
-    assert isinstance(f(x, y), type(fdiv_i_i(x, y).item()))
+    assert isinstance(f(x, y), (type(fdiv_i_i(x, y).item()), type(fdiv_i_i(x, y))))
```

Comment:
Why do we need to use the `item` method here?

Reply:
Pyccel cannot return numpy types (see #735 ) so we cannot check if the result type matches exactly because it won't. The best we can do is check that the underlying type (returned by `.item()`) matches.
We also need to check against the correct type as python returns the correct type and `isinstance(np.int8(2), int)` returns `False`
Here `type(fdiv_i_i(x, y).item())` returns `int` and `type(fdiv_i_i(x, y))` returns `np.int8`

Reply:
Thanks! I forgot that #735 is still open...